### DISCUSSION TITLE
Access with Azure Storage Explorer

### DISCUSSION THREAD

**Discussion by CaptainInler** [OvertureMaps/data#17] [Discussion]
Does anyone know how to download the files with Azure Storage Explorer?

**Comment by varapmsft** [comment]
In Azure Storage Explorer you can connect to the Overture Release blob container 'Anonymously'. Please select 'Attach to a resource' option in 'Get Started' tab, select 'ADLS Gen2 container or directory', select 'Anonymously' connection method, and then provide the release directory URL 'https://overturemapswestus2.blob.core.windows.net/release/2023-07-26-alpha.0/'. Now you should be able to browse and download data.

**Reply by bertt** [reply]
Can connect but nothing shows up... only some stats
![image](https://github.com/OvertureMaps/data/assets/538812/f05022fe-93bb-4ca9-8c65-79f9389b25c2)
![image](https://github.com/OvertureMaps/data/assets/538812/bede6e10-8d97-4f49-9eb1-bb93cf7beb5d)

**Reply by varapmsft** [reply]
Could you try upgrading to latest version of Storage Explorer? You should be seeing  the following list. I've tested this with another public user just now and they are able to browse.
![image](https://github.com/OvertureMaps/data/assets/39808840/76083f9b-bce2-4f73-8fa1-1919a0835183)

**Reply by bertt** [reply]
ah yes that was it thanks!

**Reply by CaptainInler** [reply]
Thanks @varapmsft that works.
How are this files organised? Does each file holds the data from a certain area or why is each theme splitted in 30 parquet files?
If want to load the data for only europe with Azure Storage Explorer, how do I know which file(s) I have to download?

**Reply by varapmsft** [reply]
@CaptainInler you will have to use one of the options listed in the [ReadMe ](https://github.com/OvertureMaps/data)page of the GitHub to download the data for a region. Data is partitioned into parquet files for efficient access from cloud services but they are not split by region.
